src/app/page/component/cyoa/card/card.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { Observable } from 'rxjs';
+import { PageService } from 'src/app/page/service/page-service.service';
+import {
+  CYOAPageCard,
+  Content
+} from 'src/app/services/xml-parser-service/xmp-parser.service';
+
+@Component({
+  selector: 'app-page-cyoa-card',
```

Comment:
Do you need the name `page` in the selector? I don't see the name `page` anywhere else on this file